File: frontend/Assets/Scripts/DataTransfer/Client/JoinLobbyCmd.cs
using System.IO;
using Attributes;


namespace DataTransfer.Client {

    [DTO(DTOCode.JoinLobby)]
    public class JoinLobbyCmd : ClientCommand {

        public byte LobbyId { get; private set; }


        public JoinLobbyCmd (byte lobbyId) {
            LobbyId = lobbyId;
        }


        public override void WriteMembers (BinaryWriter writer) {
            writer.Write(LobbyId);
        }

    }

}

File: backend/src/main/java/dto/cmd/client/JoinLobbyCmd.java
package dto.cmd.client;

import dto.DTOCode;
import dto.DTOs;
import io.netty.buffer.ByteBuf;


@DTOCode(DTOs.JOIN_LOBBY)
public class JoinLobbyCmd extends ClientCommand {

    @Override
    public void execute () {

    }


    @Override
    protected void writeMembers (ByteBuf byteBuf) {

    }


    @Override
    protected void readMembers (ByteBuf buffer) {

    }
}
